### PR TITLE
Change find_new_lines() and find_deleted_lines() input types

### PR DESCRIPTION
**Pull request by zurk**

### CODE REVIEW

File: lookout/core/examples/typos.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ class TyposAnalyzer(Analyzer):  # noqa: D
     description = "Reports the changes in UAST node counts."
     _log = logging.getLogger("TyposAnalyzer")
 
-    @with_changed_uasts_and_contents
+    @with_changed_uasts_and_contents(unicode=False)
```
Comment (vmarkovtsev) [inline lookout/core/examples/typos.py]: We should rather `unicode=True` and avoid manual decoding below

File: lookout/core/lib.py [hunk 2]
```diff
@@ -13,16 +13,16 @@
 from lookout.core.garbage_exclusion import GARBAGE_PATTERN
 
 
-def find_new_lines(before: File, after: File) -> List[int]:
+def find_new_lines(before: str, after: str) -> List[int]:
     """
-    Return the new line numbers from the pair of "before" and "after" files.
+    Return the new line numbers from the pair of "before" and "after" files content.
```
Comment (vmarkovtsev) [inline lookout/core/lib.py]:
```suggestion
    Return the new line numbers from the pair of "before" and "after" file contents.
```

**Comment by zurk** [comment]
Adressed

File: docs/user/getting_started.md [hunk 3]
```diff
@@ -110,9 +110,14 @@ class MyAnalyzer(Analyzer):
 analyzer_class = MyAnalyzer
 ```
 
-`@with_changed_uasts_and_contents` and `@with_uasts_and_contents` populate the corresponding
-objects with UASTs and file contents. If you need only the UASTs, use
-`@with_changed_uasts` and `@with_uasts` respectively.
+`@with_changed_uasts_and_contents(unicode=False)` and `@with_uasts_and_contents(unicode=False)` 
+populate 'changes' and 'files' objects accordingly with UASTs and file contents. `unicode` argument 
+determines if you want to get either Unicode content and UAST with Unicode positions (`True`) or 
+byte content with byte positions (`False`). In our case functions we use do not use content and
```
Comment (vmarkovtsev) [inline docs/user/getting_started.md]:
```suggestion
byte content with positions in bytes (`False`). We don't care in our case, hence set it to `False` for performance reasons.
```

File: docs/user/getting_started.md [hunk 4]
```diff
@@ -110,9 +110,14 @@ class MyAnalyzer(Analyzer):
 analyzer_class = MyAnalyzer
 ```
 
-`@with_changed_uasts_and_contents` and `@with_uasts_and_contents` populate the corresponding
-objects with UASTs and file contents. If you need only the UASTs, use
-`@with_changed_uasts` and `@with_uasts` respectively.
+`@with_changed_uasts_and_contents(unicode=False)` and `@with_uasts_and_contents(unicode=False)` 
+populate 'changes' and 'files' objects accordingly with UASTs and file contents. `unicode` argument 
+determines if you want to get either Unicode content and UAST with Unicode positions (`True`) or
```
Comment (vmarkovtsev) [inline docs/user/getting_started.md]:
```suggestion
determines whether you need Unicode content and UAST with positions in Unicode characters (`True`) or 
```

File: docs/user/getting_started.md [hunk 5]
```diff
@@ -110,9 +110,14 @@ class MyAnalyzer(Analyzer):
...
-objects with UASTs and file contents. If you need only the UASTs, use
-`@with_changed_uasts` and `@with_uasts` respectively.
+`@with_changed_uasts_and_contents(unicode=False)` and `@with_uasts_and_contents(unicode=False)` 
+populate 'changes' and 'files' objects accordingly with UASTs and file contents. `unicode` argument 
+determines if you want to get either Unicode content and UAST with Unicode positions (`True`) or 
+byte content with byte positions (`False`). In our case functions we use do not use content and 
+position information and we set `False`. Thus we keep lookout response unchanged and save 
+computational time.
+
+If you need only the UASTs, use `@with_changed_uasts` and `@with_uasts` respectively.
```
Comment (vmarkovtsev) [inline docs/user/getting_started.md]:
```suggestion
If you need only the UASTs, decorate with `@with_changed_uasts` and `@with_uasts` respectively. 
```

**Pull request merged**